README.md

```diff
@@ -421,6 +421,76 @@ chatSession.onMessage(event => {
 Subscribes an event handler that triggers whenever a message or an event (except for `application/vnd.amazonaws.connect.event.typing`) is created by any participant.
 The `data` field has the same schema as the [`Item` data type](https://docs.aws.amazon.com/connect-participant/latest/APIReference/API_Item.html) from the Amazon Connect Participant Service with the addition of the following **optional** fields: `contactId`, `initialContactId`.
 
+<span style="background-color: red">
```

Comment:
Not rendering the styling, should we remove or make "Notes:" bold?

Reply:
Thanks, updated!